src/Rings/MPolyMap/MPolyRing.jl

```diff
@@ -132,12 +132,47 @@ end
 #
 ################################################################################
 
+# Some additional methods needed for the test in the constructor for MPolyAnyMap
+is_gen(x::MPolyQuoRingElem) = is_gen(lift(x))
+is_gen(x::MPolyDecRingElem) = is_gen(forget_grading(x))
```

Comment:
Memo to self: This is not legitimate. We need to switch to some internal method like `_is_gen` for that.